src/lib/shopping-event-conversions.js

```diff
@@ -10,50 +11,93 @@ import { getUserId } from './local-storage';
 
 export type EventToFptiInputMapping = (event : Object) => FptiInput;
 
-function getStoredUserIds() : Object {
+const getDefaultFPTIEvent = () => {
```

Comment:
the name is confusing, it is not a FPTI event, we extracting some attributes. That will be used in construction of F{TI event.

Reply:
maybe `getDefaultFPTIEventAttributes`. The wording default is very obscure, I know you trying to convey a generic nature of the function, but without reading and delving into a code it is hard to understand. Maybe `fetchFPTIAttributesFromUserContext`